### ISSUE TITLE
.NET Optional Support

### ISSUE THREAD

**Issue by Luro223** [lich426/FanCtrl#31]
Hello dear developer of FanCtrl

I use WinPE based OS almost all the time and I don't have .net framework there
Is there any way to use this wonderful app without .net support because it's an issue for me
Also yeah I tried speedfan but it does not work for several old motherboards

Thanks in advance.

**Comment by regs01** [comment]
The app is made on .NET, so definitely not possible

**Comment by Luro223** [comment]
> The app is made on .NET, so definitely not possible

Yeah thanks but as I see this is only alternative with portablilty and low resource usage(except Rem0o FanControl)
SpeedFan is good but it can't control my fans from my old motherboard and issues with newer ones too
So yeah it it's not possible I will use it only on my main system not WinPE based OS
I will wait for the dev response before closing.

**Comment by lich426** [comment]
It's impossible.
All core libraries in this program are made of .net
